pkg/apis/radix/v1/radixhorizontalscalingtypes.go

```diff
@@ -0,0 +1,191 @@
+package v1
+
+import (
+	"github.com/equinor/radix-common/utils/pointers"
+	// "github.com/equinor/radix-operator/pkg/apis/deployment"
+	autoscalingv2 "k8s.io/api/autoscaling/v2"
+)
+
+const (
+	DefaultTargetCPUUtilizationPercentage = 80
+)
+
+// RadixHorizontalScaling defines configuration for horizontal pod autoscaler.
+type RadixHorizontalScaling struct {
+	// Defines the minimum number of replicas.
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	MinReplicas *int32 `json:"minReplicas,omitempty"`
+
+	// Defines the maximum number of replicas.
+	// +kubebuilder:validation:Minimum=1
+	MaxReplicas int32 `json:"maxReplicas"`
+
+	// PollingInterval configures how often to check each trigger on. Defaults to 30sec
+	PollingInterval *int32 `json:"pollingInterval"` // 30
+	// CooldownPeriod to wait after the last trigger reported active before scaling the resource back to 0. Defaults to 5min
+	CooldownPeriod *int32 `json:"cooldownPeriod"` // 300
+
+	// Defines the resource usage parameters for the horizontal pod autoscaler.
+	// +optional
+	RadixHorizontalScalingResources *RadixHorizontalScalingResources `json:"resources,omitempty"` // Deprecated: Use CPU and/or Memory triggers instead
+
+	// Defines a list of triggers the component replicas will scale on.
+	// +optional
+	Triggers *[]RadixHorizontalScalingTrigger `json:"triggers,omitempty"`
```

Comment:
```suggestion
	Triggers []RadixHorizontalScalingTrigger `json:"triggers,omitempty"`
```
No need to use array pointer

Reply:
Fixed